pkg/events/core.go

```diff
@@ -224,6 +224,23 @@ const (
 	// MaxSignalID (5499)
 )
 
+// Detector events (new detector system)
+const (
+	StartDetectorID ID = 7000
+	MaxDetectorID   ID = 7999
```

Comment:
is it intentional we are repeating the range?

```
// Signature events (extended) - defined in extended builds
	StartSignatureExtendedID ID = 7000
	MaxSignatureExtendedID   ID = 7999
```

Also, why not place it with the other ranges? https://github.com/aquasecurity/tracee/blob/main/pkg/events/core.go#L28